testsuite/features/step_definitions/command_steps.rb

```diff
@@ -624,10 +624,7 @@
 end
 
 When(/^I open avahi port on the proxy$/) do
-  sed = 's/FW_DEV_EXT=""/FW_DEV_EXT="eth0"/'
-  $proxy.run("sed -i '#{sed}' /etc/sysconfig/SuSEfirewall2")
-  sed = 's/FW_CONFIGURATIONS_EXT=""/FW_CONFIGURATIONS_EXT="avahi"/'
-  $proxy.run("sed -i '#{sed}' /etc/sysconfig/SuSEfirewall2")
+  $proxy.run('firewall-cmd --permanent --add-service=mdns')
```

Comment:
TBH I'm not sure if firewalld will manage those options
- FW_DEV_EXT="eth0" 
- FW_CONFIGURATIONS_EXT="avahi"
by itself, or we need to add some configuration for it.

Reply:
mdns service is under which avahi is known in firewalld world.
See:
/etc/sysconfig/SuSEfirewall2.d/services/avahi
```ini
## Name: Zeroconf/Bonjour Multicast DNS
## Description: Zeroconf/Bonjour Multicast DNS (mDNS) ports for Service Discovery (DNS-SD)

# Allow broadcasting on UDP port 5353.
# mdnsresponder (5354) port is not used by Avahi
UDP="mdns"
BROADCAST="mdns"
```
vs

/usr/lib/firewalld/services/mdns.xml
```xml
<?xml version="1.0" encoding="utf-8"?>
<service>
  <short>Multicast DNS (mDNS)</short>
  <description>mDNS provides the ability to use DNS programming interfaces, packet formats and operating semantics in a small network without a conventional DNS server. If you plan to use Avahi, do not disable this option.</description>
  <port protocol="udp" port="5353"/>
  <destination ipv4="224.0.0.251" ipv6="ff02::fb"/>
</service>
```

I'm not 100% sure about the eth0. In my test setup adding this to the default zone was sufficient. I'll do another test run if it is really needed or not.

Reply:
> @srbarrios I updated PR with what passes core tests suite. Please, take a look. Also do you tests proxy reboot and if it works after that?



> @srbarrios I updated PR with what passes core tests suite. Please, take a look. Also do you tests proxy reboot and if it works after that?

If you passed the testsuite in local and it works, then LGTM, just ask your room colleague @lkotek  :)